examples/workflow/1_create_model.ipynb

```diff
@@ -0,0 +1,349 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "Download this notebook :download:`here <1_create_model.ipynb>`."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step 1: Create an amplitude model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Your analysis starts by defining an amplitude module that describe the intermediate resonances in the decay process you study. For this, of course, you first have to determine what resonances there could be.\n",
```

Comment:
I think its better if we have the 80 character line break also for the rst and jupyter notebooks. Its a pain reviewing this...

Reply:
I agree, but I haven't found a Jupyter plugin that has rulers for all cell types ([ruler](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/nbextensions/ruler/readme.html) only works for code cells) nor a Jupyter notebook formatter that can do wrapping, so it's more of a pain to do this wrapping by hand.

Reply:
Added by hand in https://github.com/ComPWA/pycompwa/pull/74/commits/51b48a8618afb2997e753951df616771232a213c